dist/typings/proteus.d.ts

```diff
@@ -266,7 +266,7 @@ export declare module session {
   }
 
   abstract class PreKeyStore {
-    public prekeys: Array<number>;
+    public prekeys: Array<keys.PreKey>;
```

Comment:
Actually, allow me to expand this discussion a bit. I am thinking of a usecase where prekeys are set to expire at a time schedule (for example, one prekey everyday), rather than on a take-and-consume basis. 

I'd like to design a communication protocol which can obscure the fact that Alice and Bob are talking, and not having a central server to coordinate the prekeys. 

Here's my thinking of how such a protocol could work:

* Bob posts his prekeys publicly, say 365 of them, one per day.
* Alice grabs the prekey for today, encrypts the message, and posts to a public message hub. 
* Bob listens to the public hub and attempts to decrypt all the messages sent through it. 
* Bob would only be able to decrypt messages encrypted with his prekey for a particular day. These are the messages meant for him.

Surveillance would only be able to see that Alice is sending messages to the hub, but would have no idea who's on the receiving end.

I've looked at the code, and it looks like I need to disable automatic prekey refill and removal. Right now there's no clean way to do it... would it be possible to add hooks to the store to do this?

Maybe something like:

```
public markPrekeyForRemoval(id: number)
```